Command: make -j 10 flash
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/fus/.espressif/python_env/idf6.0_py3.13_env/lib/python3.13/site-packages/esptool/__main__.py", line 9, in <module>
    esptool._main()
    ~~~~~~~~~~~~~^^
  File "/home/fus/.espressif/python_env/idf6.0_py3.13_env/lib/python3.13/site-packages/esptool/__init__.py", line 1365, in _main
    main()
    ~~~~^^
  File "/home/fus/.espressif/python_env/idf6.0_py3.13_env/lib/python3.13/site-packages/esptool/__init__.py", line 887, in main
    esp.change_baud(args.baud)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "/home/fus/.espressif/python_env/idf6.0_py3.13_env/lib/python3.13/site-packages/esptool/targets/esp32.py", line 467, in change_baud
    ESPLoader.change_baud(self, baud)
    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "/home/fus/.espressif/python_env/idf6.0_py3.13_env/lib/python3.13/site-packages/esptool/loader.py", line 127, in inner
    return func(*args, **kwargs)
  File "/home/fus/.espressif/python_env/idf6.0_py3.13_env/lib/python3.13/site-packages/esptool/loader.py", line 1371, in change_baud
    self.flush_input()
    ~~~~~~~~~~~~~~~~^^
  File "/home/fus/.espressif/python_env/idf6.0_py3.13_env/lib/python3.13/site-packages/esptool/loader.py", line 576, in flush_input
    self._port.flushInput()
    ~~~~~~~~~~~~~~~~~~~~~^^
  File "/home/fus/.espressif/python_env/idf6.0_py3.13_env/lib/python3.13/site-packages/serial/serialutil.py", line 588, in flushInput
    self.reset_input_buffer()
    ~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/home/fus/.espressif/python_env/idf6.0_py3.13_env/lib/python3.13/site-packages/serial/serialposix.py", line 683, in reset_input_buffer
    self._reset_input_buffer()
    ~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/home/fus/.espressif/python_env/idf6.0_py3.13_env/lib/python3.13/site-packages/serial/serialposix.py", line 677, in _reset_input_buffer
    termios.tcflush(self.fd, termios.TCIFLUSH)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
termios.error: (5, 'Input/output error')
CMake Error at /home/fus/.fus/esp-idf/components/esptool_py/run_serial_tool.cmake:67 (message):
  
  /home/fus/.espressif/python_env/idf6.0_py3.13_env/bin/python;;/home/fus/.fus/esp-idf/components/esptool_py/esptool/esptool.py;--chip;esp32
  failed.



make[3]: *** [CMakeFiles/flash.dir/build.make:73: flash] Error 1
make[2]: *** [CMakeFiles/Makefile2:3115: CMakeFiles/flash.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:3122: CMakeFiles/flash.dir/rule] Error 2
make: *** [Makefile:208: flash] Error 2
